README.rst

```diff
@@ -117,6 +117,26 @@ Using dnf:
 
     $ dnf install py3status
 
+NixOS
+----------
+Add py3status as a Python 3.6 package with
```

Comment:
No love for `3.7`? Can `nixOS` haz `python2.7-py3status-3.7`, `python3.6-py3status-3.7` too?

Reply:
3.6 is the python version ;)
py3status3.7 is already available for NixOS: NixOS/nixpkgs/pull/32490

Reply:
Conflict.